code/modules/mob/living/living.dm

```diff
@@ -792,3 +793,10 @@ default behaviour is:
 		update_icons()
 	return canmove
 
+/mob/living/proc/update_water() // Involves overlays for humans.  Maybe we'll get submerged sprites for borgs in the future?
+	return
```

Comment:
Doesn't do anything yet, or is it updated to be a human specific thing elsewhere?